tests/Functional/ContentProjectRenderTest.php

```diff
@@ -143,6 +143,10 @@ public function testProjectRendering(): void {
 
     $this->drupalGet($node->toUrl());
 
+    // Assert page header - metadata.
+    $project_metadata = $this->assertSession()->elementExists('css', '.ecl-page-header');
+    $this->assertContains('Summary', $project_metadata->getText());
```

Comment:
Check the specific summary HTML element, not the whole header, also assert "Project" in its metadata HTML element, you can use CSS selector for it.